.circleci/config.yml

```diff
@@ -14,11 +14,6 @@ executors:
       - image: bitnami/redis:6.0
         environment:
           ALLOW_EMPTY_PASSWORD: "yes"
-      - image: quay.io/hmpps/hmpps-auth:latest
```

Comment:
Can we make this a fixup of the previous commit? I think in general that a change introduced and removed in the same PR should just be wiped from the commit history.

Reply:
done in c45f996